modules/apps/frontend-js/frontend-js-clay-sample-web/src/main/resources/META-INF/resources/js/App.es.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {ClayIconSpriteContext} from '@clayui/icon';
+import React from 'react';
+
+import Alert from './components/alert.es';
+
+export default () => {
+	const spritemap =
+		Liferay.ThemeDisplay.getPathThemeImages() + '/lexicon/icons.svg';
+
+	return (
+		<ClayIconSpriteContext.Provider value={spritemap}>
```

Comment:
I know you said you like your indentations, but really, we **do** wrap components in a `ClayIconSpriteContext.Provider` for you already, so you really don't need this 😂 

See https://github.com/liferay-frontend/liferay-portal/blob/master/modules/apps/frontend-js/frontend-js-react-web/src/main/resources/META-INF/resources/js/render.es.js#L72-L77

Reply:
I actually woke up in the middle of the night last night with your words ringing in my ear. I just forgot to remove it lol